environment_setup/setup_software.sh

```diff
@@ -74,8 +75,8 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
-# yaml for cfg generation (Dynamic Parameters)
-sudo apt-get install python3-yaml -y
+# Symlink qt include directory
```

Comment:
Could we indicate why we're doing this? Generally comments should indicate _why_, rather then _what_.

Reply:
fixed